managers/apps.py

```diff
@@ -3,3 +3,6 @@
 
 class ManagersConfig(AppConfig):
     name = "managers"
+
+    def ready(self) -> None:
+        import managers.signals  # noqa, flake8 F401 issue  # pylint: disable=unused-import
```

Comment:
Why do you need to import it this way?

Reply:
I couldn't import TaskActivityType in manager app in models so I decided to refactor whole handlers to file `signals.py`. On this page, It is written that we have to do this because if I wouldn't my handlers will not catch signals.
https://simpleisbetterthancomplex.com/tutorial/2016/07/28/how-to-create-django-signals.html

Reply:
What was the exact problem with imports?

Reply:
I hade this exception 
```
Traceback (most recent call last):
  File "manage.py", line 19, in <module>
    execute_from_command_line(sys.argv)
  File "/home/szymi/virtualenv/sheetstorm-SgY9UmI-/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/szymi/virtualenv/sheetstorm-SgY9UmI-/lib/python3.7/site-packages/django/core/management/__init__.py", line 357, in execute
    django.setup()
  File "/home/szymi/virtualenv/sheetstorm-SgY9UmI-/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/szymi/virtualenv/sheetstorm-SgY9UmI-/lib/python3.7/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/home/szymi/virtualenv/sheetstorm-SgY9UmI-/lib/python3.7/site-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/home/szymi/virtualenv/sheetstorm-SgY9UmI-/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/szymi/Dokumenty/GitRepos/sheetstorm/sheetstorm/managers/models.py", line 9, in <module>
    from employees.models import TaskActivityType
  File "/home/szymi/Dokumenty/GitRepos/sheetstorm/sheetstorm/employees/models.py", line 14, in <module>
    from managers.models import Project
ImportError: cannot import name 'Project' from 'managers.models' (/home/szymi/Dokumenty/GitRepos/sheetstorm/sheetstorm/managers/models.py)
```
I think it happen because We imported Project in `employees app` and in `managers app` models we want to import `TaskActivityType` which is needed to work with handlers.

Reply:
@rwrzesien When I make import locally it works fine

Reply:
I see that you have resolved this by making import directly in function, so it works now right?

Reply:
@rwrzesien , Yes it works but pylint gives one warning about `cycling import`